### DISCUSSION TITLE
Losing logs after update

### DISCUSSION THREAD

**Discussion by KS5Y** [mbridak/not1mm#94] [General]
Hi folks,
Quick question, what should be the default location for logs? 

Every time that I run the following command:
pip install -U not1mm

I'm losing logs, I don't know if the folder is being overwritten or recreated
I guess I should backup those files.

thank you

**Comment by mbridak** [comment]
Hi, Are you using Linux?

 If So:
### Data

If your system has an `XDG_DATA_HOME` environment variable set, the database and
CW macro files can be found there. Otherwise they will be found at
`yourhome/.local/share/not1mm`

### Config

Configuration file(s) can be found at the location defined by `XDG_CONFIG_HOME`.
Otherwise they will be found at `yourhome/.config/not1mm`

### Caveat
But, I had accepted a PR which had code that moved them to `~/.not1mm/` for some reason. I didn't notice till I had people messaging me that their logs were gone. So I Yanked that version, fixed and re-uploaded to PyPI. You may have been running the version I Yanked. That's my guess.

**Reply by KS5Y** [reply]
Hi @mbridak 
I'm using FreeBSD 13.3 but I found out that logs are placed in /home/user/.local/share/not1mm

I'm going to backup my files before any update to prevent issues.

Thank you!

**Reply by mbridak** [reply]
Yes, backing up is always a great idea.
This thing is Beta. Mainly as this absolves me of many coding and release sins.
Cheers!
